tests/functional/UserTest.php

```diff
@@ -115,6 +115,12 @@ public function testUserLogin()
 
     public function testUserCancel()
     {
+        // TODO: re-enable this test once the Drupal Code Generator is
+        // compatible with Drupal 9.
+        if (version_compare('9.0.0', \Drupal::VERSION, '>=')) {
+            $this->markTestSkipped('Drupal Code Generator not working on Drupal 9 yet.');
```

Comment:
@Chi-teck Could you update DCG to add `core_version_requirement` to generated module info.yml files please?

Reply:
Added in 1.30.5.